### PR TITLE
fix(autoware_pointcloud_preprocessor): fix variableScope

### PR DESCRIPTION
**Pull request by kobayu858**
## Description
This is a fix based on cppcheck variableScope warnings

```
sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp:222:10: style: The scope of the variable 'global_point_stamp' can be reduced. [variableScope]
  double global_point_stamp;
         ^
```
## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp [hunk 1]
```diff
@@ -217,13 +217,12 @@ void DistortionCorrector<T>::undistortPointCloud(
...
   bool is_twist_time_stamp_too_late = false;
   bool is_imu_time_stamp_too_late = false;
-  double global_point_stamp;
 
   for (; it_x != it_x.end(); ++it_x, ++it_y, ++it_z, ++it_time_stamp) {
     bool is_twist_valid = true;
     bool is_imu_valid = true;
 
-    global_point_stamp =
+    double global_point_stamp =
```
Comment (Shin-kyoto) [inline sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp:225]:
```suggestion
    const double global_point_stamp =
```

**Review by Shin-kyoto — CHANGES_REQUESTED** [review]

File: sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp (hunk 1 above)
Comment (kobayu858) [reply sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp]: Thank you for pointing this out. I have fixed it.

**Review by Shin-kyoto — APPROVED** [review]
LGTM

I confirmed that this PR's content is good to merge.
I did NOT check this PR by running autoware.

**Comment by Shin-kyoto** [comment]
@vividf 
Can you review this PR?

**Review by vividf — COMMENTED** [review]
@kobayu858 Thanks!
My only consideration is the execution time, could you do a simple analysis of the execution time?

If the time is negligible, then we can merge this.

**Review by vividf — APPROVED** [review]
LGTM!

**Pull request merged**